### PR TITLE
610 connection pads

### PR DESCRIPTION
**Pull request by grace-harper-ibm**
### What are the issues this pull addresses (issue numbers / links)?
#610 

### Did you add tests to cover your changes (yes/no)?
no
### Did you update the documentation accordingly (yes/no)?
no
### Did you read the CONTRIBUTING document (yes/no)?
no
### Summary
Param Entry Window now has a default connection pad for qubits to make it easier for users to know how to add connection pads

### CODE REVIEW

**Review by JeremyDrysdale-ibm — CHANGES_REQUESTED** [review]
Please merge main and resolve the merge conflicts.

File: qiskit_metal/_gui/widgets/create_component_window/parameter_entry_window.py [hunk 1]
```diff
@@ -310,18 +310,48 @@ def generate_model_data(self):
...
+            OPTIONS = 'options'
+            # deepcopy so instance's options aren't pointing
+            # to same dict as template_options nor default_options
             copied_options = copy.deepcopy(options)
-            param_dict['options'] = copied_options
+            param_dict[OPTIONS] = copied_options
+
+            # custom options for connection_pads
+            def_opt_dict_to_replace = {
+                '_default_connection_pads': ('connection_pads', {
```
[the remark this answers is not in the document]
Comment (grace-harper-ibm) [reply qiskit_metal/_gui/widgets/create_component_window/parameter_entry_window.py:334]: Do you want all the default options shown? I can do that, but earlier there had been talk that this was too many options for the user to see when they opened the tab.

[the remark this answers is not in the document]
Comment (zlatko-minev) [reply qiskit_metal/_gui/widgets/create_component_window/parameter_entry_window.py:334]: It's a good question. 
I was thinking here about the fact that in the following code we are presuming to know the keys and values that will necessarily exist for a connection pad dictionary, which may or may not be the same for different qubits. If the component doesn’t use loc_W or loc_H for example, these params would not function. 

I was wondering if we could make it more general, by not restricting the pad keys to ones that the component may have or may not have. 
You're right the tradeoff for this generality would then be to potentially have too many parameters. 
I may in this case lean on the side of having more parameters, just to be more general in this case, and I am open to comments

[the remark this answers is not in the document]
Comment (grace-harper-ibm) [reply qiskit_metal/_gui/widgets/create_component_window/parameter_entry_window.py]: I've updated the code

**Review by JeremyDrysdale-ibm — APPROVED** [review]

**Review by zlatko-minev — APPROVED** [review]
Nice!

**Pull request merged**